alpha/lib/model/LiveEntryServerNode.php

```diff
@@ -111,7 +110,13 @@ public function postDelete(PropelPDO $con = null)
 			/* @var $liveEntry LiveEntry */
 			$entryServerNodes = EntryServerNodePeer::retrievePlayableByEntryId($liveEntry->getId());
```

Comment:
Since the 2 files have duplicate code #111 -> #126, consider moving the code to a single function that will be called from the 2 places